__tests__/fixtures1.blocks/BlockWithDeclaredMix/BlockWithDeclaredMix.js

```diff
@@ -0,0 +1,8 @@
+import { decl } from 'bem-react-core';
+
+export default decl({
+    block : 'BlockWithDeclaredMix',
+    mix() {
```

Comment:
@tadatuta what about `...this.__base(...arguments)`? I understand what you mean, but this helper isn't connected to this issue. If want make similar helpers we need to have: `addMix`, `addAttrs`, `addCls`, `addMods`, `addStyle` and etc.